LocalizationUtility/LocalizationAPI.cs

```diff
@@ -8,10 +8,14 @@ public class LocalizationAPI : ILocalizationAPI
         public void RegisterLanguage(ModBehaviour mod, string name, string translationPath) 
             => LocalizationUtility.Instance.RegisterLanguage(mod, name, translationPath, TextTranslation.Language.ENGLISH);
         public void RegisterLanguage(ModBehaviour mod, string name, string translationPath, string languageToReplace)
-            => LocalizationUtility.Instance.RegisterLanguage(mod, name, translationPath, (TextTranslation.Language) Enum.Parse(typeof(TextTranslation.Language), languageToReplace));
+            => LocalizationUtility.Instance.RegisterLanguage(mod, name, translationPath, (TextTranslation.Language) Enum.Parse(typeof(TextTranslation.Language), languageToReplace, true));
```

Comment:
...pardon a noob question, but is this correct? Doesn't think nuke the `AddLanguageFont` functionality?

Reply:
I don't understand what you are asking. The "ignore case" means that it will find the language even if `languageToReplace` is lowercase.

Reply:
This... is not the diff I'd seen. My apologies; please disregard.